migrations/v9-MIGRATION.md

```diff
@@ -0,0 +1,25 @@
+# V9 API Migration Guide
+
+This release updates the SDK to use Google Play Billing Library 8. This version of the Billing Library removed APIs to query for expired subscriptions and consumed one-time products, aside from other improvements. You can check the full list of changes here: https://developer.android.com/google/play/billing/release-notes#8-0-0
+
+Additionally, we've also updated Kotlin to 2.0.21 and our new minimum version is Kotlin 1.8.0+. If you were using an older version of Kotlin, you will need to update it.
+
+Regarding API changes, we've also removed data classes from our public APIs. This means that for classes that were previously data classes, the `copy` function and `componentN` functions (destructuring declarations) have been removed. `equals` and `hashCode` functions still work as before.
+
+### Play Billing Library 8: No expired subscriptions or consumed one-time products
+
+**Note:** the following is only relevant if you recently integrated RevenueCat, and do not (yet) have all your transactions imported. 
```

Comment:
I'm late to the game here, but what is "recently" in this context? this migration guide will live for a long time. Maybe we can tie it to a real-life date, even if arbitrary?

Reply:
Right, I don't think we can tie it to a specific date unfortunately, though it will become less relevant over time. I think @JayShortway's suggestion makes sense! Will make a PR with it.